Dockerfile

```diff
@@ -50,6 +50,11 @@ RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
 ENV LIBRARY_PATH /usr/local/lib:$LIBRARY_PATH
 ENV LD_LIBRARY_PATH /usr/local/lib:$LD_LIBRARY_PATH
 
+# Install protoc
+RUN curl -LO "https://github.com/protocolbuffers/protobuf/releases/download/v3.15.8/protoc-3.15.8-linux-x86_64.zip"
```

Comment:
Why need this?

Reply:
I don't know, apparently it's needed after https://github.com/tikv/tikv/pull/12430.

Reply:
/cc @YuJuncen

Reply:
Because #12430 imports `tonic` which uses `prost` as default (it seems we cannot trivially replace `prost` with `protobuf`), and `prost` doesn't bundle `protoc` any more after some version.

Perhaps we'd better get rid of `prost` in the future or port `protobuf-rust` to `tonic`.  The progress could be tracked in https://github.com/tikv/tikv/issues/13867.